pwnlib/heap/glmalloc/utils.py

```diff
@@ -0,0 +1,33 @@
+import re
+from pwnlib.util import packing
+
+
+def u32(bytes_):
+    return packing.u32(bytes_)
+
+
+def u64(bytes_):
+    return packing.u64(bytes_)
+
+
+def p32(number):
+    return packing.p32(number)
+
+
+def p64(number):
+    return packing.p64(number)
+
+
+def align_address(address, align):
+    """Align the address to the given size."""
+    return address + ((align - (address % align)) % align)
+
+
+def get_libc_version_from_name(name):
+    libc_version = tuple(int(_) for _ in re.search(r"libc6?[-_](\d+)\.(\d+)\.so", name).groups())
```

Comment:
This needs to handle failure cases better, and should be split into multiple lines.  If there's no match, `.groups()` will throw an exception.  This should handle failure gracefully.

Reply:
Done in 32e0e1f